recipes/looptools/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "looptools" %}
+{% set version = "2.16" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://feynarts.de/looptools/LoopTools-{{ version }}.tar.gz
+  sha256: 4113467575db3a14405d62d9e516b3b90410b73ea8d20c8eb8d70a30fc5cc9cb
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}-static
+
+    build:
+      skip: true  # [win]
+      run_exports:
+        - {{ pin_subpackage('looptools-static', max_pin='x.x') }}
+      ignore_run_exports_from:
+        - {{ compiler('cxx') }}
+      script:
+        # FIXME: Avoid error from trying to create directories that exist
+        - sed -i 's/mkdir /mkdir -p /g' makefile.in
+        - ./configure --help
+        # FIXME: Author has disabled builds with dynamic linking
+        # c.f. https://github.com/hep-packaging-coordination/packaging-hep-simulation-stack/issues/4#issuecomment-2489198926
+        # - ./configure --prefix=$PREFIX --dynamic --64 --native
+        - ./configure --prefix=$PREFIX --64  # [build_platform != target_platform]
+        - ./configure --prefix=$PREFIX --64 --native  # [build_platform == target_platform]
```

```suggestion
        - ./configure --prefix=$PREFIX --64 --native  # [build_platform == target_platform]
```

Reviewers, is setting `-march=native` here safe to do for a CI job? Or should this only ever be done when you're building on the same machine you plan to run on?

---

Never ever set `--native` if you want to use the binary on multiple system. Please use the provided C(XX)FLAGS from the compiler activation scripts.